tests/Integration/AbstractIntegrationTest.php

```diff
@@ -48,7 +48,7 @@ protected function getPackageProviders($app): array
 
     protected function defineDatabaseMigrations(): void
     {
-        $this->loadMigrationsFrom(static::getLaravelPath('Database' . DIRECTORY_SEPARATOR . 'migrations'));
+        $this->loadMigrationsFrom(self::getLaravelPath('Database' . DIRECTORY_SEPARATOR . 'migrations'));
```

Comment:
This essentially causes `getLaravelPath` to not be overrulable, is that a conscious choice?